doc/datastorage_setup.md

```diff
@@ -0,0 +1,53 @@
+# ZeroMQ installation on Linux
+
+## Quick start ##
+This section provides how to download and install zeromq
+
+#### 1. Install the dependency packages
+- sudo apt-get install libtool pkg-config build-essential autoconf   automake
```

Comment:
Better to apply the coding style font like `sudo apt-get...`.

Reply:
Done